content/build/cloud/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Docker Cloud Build
+description: Get started with Docker Cloud Build
+keywords: build, cloud, cloud build, remote builder
+aliases:
+  - /hydrobuild/
+---
+
+Docker Cloud Build is a service that lets you build your container images
+faster, both locally and in CI. Builds run on cloud infrastructure optimally
+dimensioned for your workloads, no configuration required. The service uses a
+remote build cache, ensuring fast builds anywhere and for all team members.
+
+## How Docker Cloud Build works
+
+Using Docker Cloud Build is no different from running a regular build. You invoke a
+build the same way you normally would, using `docker buildx build`. The
+difference is in where and how that build gets executed.
+
+By default when you invoke a build command, the build runs on a local instance
+of BuildKit, bundled with the Docker daemon. With Docker Cloud Build, you send the
+build request to a BuildKit instance running remotely, in the cloud.
```

Comment:
Should we add that we send the build request in a fully secured manner ? Maybe also that the builders are not shared with others ?

Reply:
we have the "not shared with others" bit in the FAQ. I plan to incorporate the FAQ page into the main docs at some point but it's there for now. I'll add the "fully secure" aspect though!